package.json

```diff
@@ -23,16 +24,19 @@
   },
   "homepage": "https://github.com/singnet/alpha-blockchain#readme",
   "dependencies": {
+    "moment": "^2.22.2",
```

Comment:
what is this used for

Reply:
gas analysis stuff. this really should be in devDependencies. will move it.